packages/fxa-settings/src/models/Account.ts

```diff
@@ -729,12 +700,13 @@ export class Account implements AccountData {
             accountResetToken,
             email,
             newPassword,
-            options: { sessionToken: true },
+            options: { sessionToken: true, keys: true },
           },
         },
       });
       currentAccount(getOldSettingsData(accountReset));
       sessionToken(accountReset.sessionToken);
+      return accountReset;
```

Comment:
Probably a follow up item, but it'd be great to return something other than 'any' here.